guides/code-signing/code-signing-windows.md

```diff
@@ -56,3 +58,133 @@ module.exports = {
 };
 ```
 {% endcode %}
+
+## Using Azure Trusted Signing
+
+[Azure Trusted Signing](https://azure.microsoft.com/en-us/products/trusted-signing) is Microsoft's modern cloud-based alternative to EV certificates. It is the cheapest option for code signing on Windows, and it gets rid of SmartScreen warnings.
+
+As of November 2024, Azure Trusted Signing is available to organizations with 3+ years of verifiable business history and to individuals. Microsoft is [looking to make the program available](https://github.com/Azure/trusted-signing-action/issues/42#issuecomment-2488402061) to organizations with a shorter history, too. If you're reading this at a later point, it could make sense to check.
+
+### Prerequisites <a href="#prerequisites-trusted-signing" id="prerequisites-trusted-signing"></a>
+
+First, create an Azure account and set up Azure Trusted Signing in your account as described [here](https://melatonin.dev/blog/code-signing-on-windows-with-azure-trusted-signing/).
+
+Then install the dependencies for local code signing as described [here](https://melatonin.dev/blog/code-signing-on-windows-with-azure-trusted-signing/#step-8-signing-locally). Also create the required `metadata.json` file in an arbitrary location on your computer.
+
+### Configuring Electron Forge <a href="#configuring-forge-trusted-signing" id="configuring-forge-trusted-signing"></a>
+
+#### Installing npm dependencies
+
+In your project directory, do the following:
+
+1. Install the `dotenv-cli` package: `npm i -D dotenv-cli`
```

Comment:
Can leave this unaddressed if this is too bike-sheddy but I think you can use  `dotenv` directly with the preload option:

https://github.com/motdotla/dotenv?tab=readme-ov-file#preload

Might be worth considering because I think `dotenv-cli` depends on `dotenv` anyways.

Reply:
I think that makes sense. Happy to keep it as-is.